CV.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title></title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="style-CV.css" />
+    <script src="https://kit.fontawesome.com/990c8adfeb.js" crossorigin="anonymous"></script>
+    <title>CV Misha Labruyère</title>
+</head>
+
+<body>
+    <div id="miseenpage">
+        <div id="apropos">
+
+            <img src="photocv.JPG">
+
+            <div class="contact">
+                <h3>Contact</h3>
+            </div>
+
+            <ul>
+                <h4>Adresse</h4>
+                18 rue d'Alsace
+                78100 Saint Germain en Laye
+                <h4>Téléphone</h4>
+                06 10 28 05 22
+                <h4>Email</h4>
+                mishalabruyere@gmail.com
+            </ul>
+
+            <div class="compétences">
+                <h3>Compétences</h3>
+            </div>
+
+            <ul>
+                <li>Photoshop</li>
+                <li>Lightroom</li>
+                <li>Maitrise des lumières</li>
+            </ul>
+
+            <div class="loisirs">
+                <h3>Loisirs</h3>
+            </div>
+
+            <ul>
+                <li>Photographie</li>
+                <li>Cinema</li>
+                <li>Sport (Fitness)</li>
+                <li>Voyages</li>
+                <li>Arts</li>
+            </ul>
+
+        </div>
+        <div id="parcours">
+
+            <header>
+                <h1>Misha</br>Labruyère</h1>
+            </header>
+
+            <div>
+
+                <h2><i class="fas fa-info-circle"></i> A propos</h2>
```

Comment:
Essaye de rendre tes icônes accessibles en les masquant si elles sont décoratives ou donner un texte alternatif si sémantiques

Reply:
https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility